File: Operators-And-Expressions/Trapezoids/TrapezoidArea.cs
// Problem 9. Trapezoids

// Write an expression that calculates trapezoid's area by given sides a and b and height h.

using System;

class TrapezoidArea
{
    static void Main()
    {
        Console.WriteLine("Insert trapezoid side a: ");
        double a = double.Parse(Console.ReadLine());
        Console.WriteLine("Insert trapezoid side b: ");
        double b = double.Parse(Console.ReadLine());
        Console.WriteLine("Insert trapezoid height: ");
        double h = double.Parse(Console.ReadLine());
        Console.WriteLine("Trapezoid area: " + ((a+b)/2)*h);
    }
}

File: JavaScript Fundamentals Homeworks/Operators and Expressions/trapezoid-area.js
// Problem 8. Trapezoid area
// Write an expression that calculates trapezoid's area by given sides a and b and height h.

var a = 8.5, b = 4.3, h = 2.7;

console.log(trapezoidArea(a, b, h));

function trapezoidArea(a, b, h){
    return ((a + b) / 2) * h;
}